pinot-common/src/main/java/org/apache/pinot/common/request/context/FilterContext.java

```diff
@@ -116,4 +116,8 @@ public String toString() {
         throw new IllegalStateException();
     }
   }
+
+  public String getResultColumnName() {
```

Comment:
Suggest adding a util method in `AggregationFunctionUtils.getResultColumnName(AggregationFunction aggregationFunction, @Nullable FilterContext filter)` to directly return the full result column name instead of adding it here. Also suggest making `FILTER(WHERE` capitalized since they are preserved key word